### PR TITLE
Fix #1360 inputs tooltip on RTL locales

### PR DESCRIPTION
**Pull request by mohamed7afezz**
The tooltip should be float right to beginning of the write in Email and password input.

### CODE REVIEW

File: app/styles/modules/_tooltip.scss [hunk 1]
```diff
@@ -2,7 +2,12 @@
   background: $error-background-color;
   border-radius: $small-border-radius;
   color: $message-text-color;
-  left: 3px;
+  html[dir=ltr] & {
+  	left: 3px;
+  	}
```
Comment (shane-tomlinson) [inline app/styles/modules/_tooltip.scss]: Can you fix the indentation here? The closing brace should be left aligned with the selector. a two space indent, and a blank line above the entire declaration:

```css
color: $message-text-color;
html[dir=ltr] & {
  left: 3px;
}
```

You can see where there are issues by running `grunt sasslint` at the command line from the root directory.

**Comment by shane-tomlinson** [comment]
Thanks for the continued series of pull requests @mohamed7afezz! Left one comment about styling, otherwise the functionality is great!

screen shot 2016-05-26 at 10 13 03

**Comment by mohamed7afezz** [comment]
Yup! it is good for me :+1:

File: app/styles/modules/_tooltip.scss (hunk 1 above)
Comment (vladikoff) [inline app/styles/modules/_tooltip.scss]: @mohamed7afezz see above ^

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3381b6a153559045da23f766ae5942ab219dba31
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 1a4b9f7fbb2ae6452aad875a077e6741d3ad87c3
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by mohamed7afezz** [comment]
plz check it now @shane-tomlinson @vladikoff

**Comment by vladikoff** [comment]
Thank you!

**Pull request merged**